src/atoms/author/author.tsx

```diff
@@ -46,4 +54,48 @@ const Author = ({ authors, prepend }: AuthorProps) => {
   );
 };
 
+/**
+ *
+ * ### How to pass custom Author component from the app?
+ * ...
+ * ...
+ * ampRoutes(app, {
+ *  ...
+ *  ...
+ *  render: {
+ *    authorCardRender: ({ story, config, storyType, theme }) => {
+ *     const authorSettings =
+ *         get(
+ *           config,
+ *           ["additionalConfig", "story", `${camelCase(storyType)}-story`, "settings", "authorDetails"],
+ *           {}
+ *         ) || {};
+ *       const authorStyle = get(authorSettings, ["template"], "default");
+ *       if (authorStyle !== "default") return null;
+ *       const { enableLocalization = false, localizedElements = {} } = get(
+ *         config,
+ *         ["additionalConfig", "general", "localization"],
+ *         {}
+ *       );
+ *       const localizedElementData = enableLocalization ? localizedElements : {};
+ *       const { buttonLabels = {} } = localizedElementData;
+ *       const { authorLabel: localizedAuthorLabel, guestAuthorLabel: localizedGuestAuthorLabel } = buttonLabels;
+ *       const authorConfig = {
+ *         ...authorSettings,
+ *         localizedAuthorLabel,
+ *         localizedGuestAuthorLabel,
+ *       };
+ *       return <AuthorCard authors={story.authors} config={authorConfig} theme={theme} />;
+ *     }
+ *  },
+ * }
+ *  ...
+ *  ...
+ * ```
+ *
+ * @category Molecules
+ * @module Author
+ * @component
+ */
+
```

Comment:
Instead of adding documentation here, pls make a note of author card render under `renders` in the tutorial [here](https://github.com/quintype/quintype-amp/blob/master/docs-src/tutorials/opts.md)

Reply:
Done